src/components/accessibility/screen_reader_only/screen_reader_only.styles.ts

```diff
@@ -8,13 +8,29 @@
 
 import { css } from '@emotion/react';
 
-export const euiScreenReaderOnlyStyles = () => {
-  return css`
+/*
+ * Mixin
+ */
```

Comment:
Same as above, it's worth noting this context given the # of times we've encountered bugs around SR only CSS now
```suggestion
/*
 * Mixin that hides elements offscreen to only be read by screen reader
 * See https://github.com/elastic/eui/pull/5130 and https://github.com/elastic/eui/pull/5152 for more info
 */
```